morf-postgresql/src/main/java/org/alfasoftware/morf/jdbc/postgresql/PostgreSQLDialect.java

```diff
@@ -713,13 +714,10 @@ public Collection<String> alterTableChangeColumnStatements(Table table, Column o
   private String addAlterTableConstraint(Table table, Column newColumn, boolean alterNullable, boolean alterType,
       boolean alterDefaultValue) {
     StringBuilder sqlBuilder = new StringBuilder();
-    sqlBuilder.append("ALTER TABLE " + schemaNamePrefix(table) + table.getName()
-                + (alterNullable ? " ALTER COLUMN " + newColumn.getName() + (newColumn.isNullable() ? " DROP NOT NULL" : " SET NOT NULL") : "")
-                + (alterNullable && alterType ? "," : "")
-                + (alterType ? " ALTER COLUMN " + newColumn.getName() + " TYPE " + sqlRepresentationOfColumnType(newColumn, false, false, true) : "")
-                + (alterDefaultValue && (alterNullable || alterType) ? "," : "")
-                + (alterDefaultValue ? " ALTER COLUMN " + newColumn.getName() + (!newColumn.getDefaultValue().isEmpty() ? " SET DEFAULT " + sqlForDefaultClauseLiteral(newColumn) : " DROP DEFAULT") : "")
-        );
+    sqlBuilder.append("ALTER TABLE ").append(schemaNamePrefix(table)).append(table.getName())
+            .append(alterNullable ? " ALTER COLUMN " + newColumn.getName() + (newColumn.isNullable() ? " DROP NOT NULL" : " SET NOT NULL") : "")
+            .append(alterNullable && alterType ? "," : "").append(alterType ? " ALTER COLUMN " + newColumn.getName() + " TYPE " + sqlRepresentationOfColumnType(newColumn, false, false, true) : "")
+            .append(alterDefaultValue && (alterNullable || alterType) ? "," : "").append(alterDefaultValue ? " ALTER COLUMN " + newColumn.getName() + (!newColumn.getDefaultValue().isEmpty() ? " SET DEFAULT " + sqlForDefaultClauseLiteral(newColumn) : " DROP DEFAULT") : "");
     return sqlBuilder.toString();

```

Comment:
I think we should revert this change, and drop the `StringBuilder` completely instead.
The mix of appends with conditional concats inside is probably worse than before.

Reply:
Done.